packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -180,18 +198,75 @@ void main() {
     const ResolutionPreset testResolutionPreset = ResolutionPreset.veryHigh;
     const bool enableAudio = true;
     const int testSurfaceTextureId = 6;
+
+    // Mock/Detached objects for (typically attached) objects created by
+    // createCamera.
+    final MockProcessCameraProvider mockProcessCameraProvider =
+        MockProcessCameraProvider();
+    final MockPreview mockPreview = MockPreview();
+    final MockCameraSelector mockBackCameraSelector = MockCameraSelector();
+    final MockImageCapture mockImageCapture = MockImageCapture();
+    final MockImageAnalysis mockImageAnalysis = MockImageAnalysis();
+    final MockRecorder mockRecorder = MockRecorder();
+    final MockVideoCapture mockVideoCapture = MockVideoCapture();
     final MockCamera mockCamera = MockCamera();
     final MockCameraInfo mockCameraInfo = MockCameraInfo();
     final MockLiveCameraState mockLiveCameraState = MockLiveCameraState();
+    bool cameraPermissionsRequested = false;
+    bool startedListeningForDeviceOrientationChanges = false;
+
+    // Tell plugin to create mock/detached objects and stub method calls for the
+    // testing of createCamera.
+    camera.proxy = CameraXProxy(
```

Comment:
It looks like you are setting default values in multiple tests. You could probably create a `TestCameraXProxy` or create a method that sets the default mock value classes.